x-pack/solutions/search/plugins/search_playground/server/utils/context_document_mapper.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { Document } from '@langchain/core/documents';
+import type { SearchHit } from '@elastic/elasticsearch/lib/api/types';
+import type { ElasticsearchRetrieverContentField } from '../types';
+import { getValueForSelectedField } from './get_value_for_selected_field';
+
+export const contextDocumentHitMapper =
+  (contentField: ElasticsearchRetrieverContentField) =>
+  (hit: SearchHit): Document => {
+    let pageContent: string = '';
+    const makePageContentForField = (field: string) =>
+      `${field}: ${getValueForSelectedField(hit, field)}`;
```

Comment:
update this function to check if contentField has a field value

Reply:
thanks!